assets/_scss/elements/_typography.scss

```diff
@@ -118,7 +118,8 @@ dfn {
 
 // Custom typography
 
-p {
+p, 
+.usa-content li {
```

Comment:
Can we use `usa-content-list` instead? It will be a less global rule and that seems like a good use case.